ansible/roles/vm_set/tasks/start_k8s_vm.yml

```diff
@@ -0,0 +1,69 @@
+- name: Create directory for Ubuntu VM disks
+  file: path={{ k8s_root_path }}/{{ vm_name }} state=directory mode=0755
+
+- name: Check destination file existence
+  stat: path={{ disk_image }}
```

Comment:
>disk_image [](start = 16, length = 10)

This path is not relevant to below tasks. Typo?

Reply:
I removed many `when` checks, just left the one so that the disk image would not get resized twice